src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1349,7 +1351,7 @@ public String getFeedbackSessionResultSummaryAsCsv(String courseId,
      */
     public String getFeedbackSessionResultSummaryInSectionAsCsv(
             String courseId, String feedbackSessionName, String instructorEmail,
-            String section, String filterText, boolean isMissingResponsesShown, boolean isStatsShown)
+            String section, String filterText, boolean isMissingResponsesShown, boolean isStatsShown, String questionNumber)
```

Comment:
questionId is different from questionNumber, I think questionId is the database key whereas questionNumber is the number.